commit 3d288350b27fd7e578fb1c033c92760645a07528 (refs/changes/96/2796/3)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2017-10-27T22:16:40+00:00 (1 year, 11 months ago)
    
    TOOLS-1894 Branch protection too strict
    Reviewed by: Trent Mick <trent.mick@joyent.com>
    Approved by: Trent Mick <trent.mick@joyent.com>

diff --git a/bin/crimport b/bin/crimport
index 1608ba2..5831976 100755
--- a/bin/crimport
+++ b/bin/crimport
@@ -287,14 +287,7 @@ function igp_github_protect
 		        "strict": true,
 		        "contexts": [ ]
 		    },
-		    "required_pull_request_reviews": {
-		        "dismissal_restrictions": {
-		            "users": [],
-		            "teams": []
-		        },
-		        "dismiss_stale_reviews": true,
-		        "require_code_owner_reviews": true
-		    },
+		    "required_pull_request_reviews": null,
 		    "enforce_admins": true,
 		    "restrictions": {
 		        "users": [
